hw/ip/pwm/lint/pwm.vbl

```diff
@@ -0,0 +1,6 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# These lines are too long due to generated code
```

Comment:
As a note: if we can simplify the generated reg-top as I'm suggesting then we won't need this waiver entry.